src/Facebook/FacebookBatchRequest.php

```diff
@@ -62,20 +62,21 @@ public function __construct(FacebookApp $app = null, array $requests = [], $acce
     }
 
     /**
-     * A a new request to the array.
+     * Adds a new request to the array.
      *
      * @param FacebookRequest|array $request
      * @param string|null           $name
+     * @param string[]              $options The batch options e.g. 'omit_response_on_success'
      *
      * @return FacebookBatchRequest
      *
      * @throws \InvalidArgumentException
      */
-    public function add($request, $name = null)
+    public function add($request, $name = null, array $options = [])
```

Comment:
So do we merge this to master and start a new PR for 5.4 with the aforementioned interface, or do we discard this PR and do just the PR for 5.4?

Also can I rename this to `Add full batch support` ?